apps/admin-portal/src/components/identity-providers/forms/index.ts

```diff
@@ -18,5 +18,5 @@
 
 export * from "./general-details-form";
 export * from "./advanced-configurations-form";
-export * from "./authenticator-form-factory";
+export * from "./factories/authenticator-form-factory";
```

Comment:
Shall we add an index to `factories`

Reply:
Fixed in cb34564693f1d730c6a791813685e95dd490c094.